docs/design.md

```diff
@@ -1,13 +1,12 @@
----
-title: GPJax Principles
----
+# GPJax Principles
 
 GPJax is designed to be a Gaussian process package that provides an
 accurate representation of the underlying maths. Variable names are
-designed to closely, if not exactly, match the notation in the <strong data-cite="rasmussen2006gaussian"></strong>. We here list the notation used
-in GPJax with its corresponding mathematical quantity.
+designed to closely, if not exactly, match the notation in
+<strong data-cite="rasmussen2006gaussian"></strong>.
```

Comment:
Does `<strong data-cite="rasmussen2006gaussian"></strong>` need fixing to `<strong data-cite=\"rasmussen2006gaussian\"></strong>`?

(i.e., need a `\` before each `"`).